AnkiDroid/src/main/java/com/ichi2/ui/SeekBarPreference.java

```diff
@@ -44,6 +47,8 @@ public SeekBarPreference(Context context, AttributeSet attrs) {
         mMax = attrs.getAttributeIntValue(androidns, "max", 100);
         mMin = attrs.getAttributeIntValue(AnkiDroidApp.XML_CUSTOM_NAMESPACE, "min", 0);
         mInterval = attrs.getAttributeIntValue(AnkiDroidApp.XML_CUSTOM_NAMESPACE, "interval", 1);
+        mXLabel = attrs.getAttributeValue(AnkiDroidApp.XML_CUSTOM_NAMESPACE, "xlabel");
+        mYLabel = attrs.getAttributeValue(AnkiDroidApp.XML_CUSTOM_NAMESPACE, "ylabel");
```

Comment:
Note: This would be better implemented as stated: https://stackoverflow.com/questions/46689210/android-how-to-use-string-resource-from-xml-file-in-property-of-customized-pref

```suggestion
        mXLabel = attrs.getAttributeResourceValue(AnkiDroidApp.XML_CUSTOM_NAMESPACE, "xlabel", 0);
        mYLabel = attrs.getAttributeResourceValue(AnkiDroidApp.XML_CUSTOM_NAMESPACE, "ylabel", 0);
```